src/components/MarkdownProvider/components/OverviewLinks.tsx

```diff
@@ -0,0 +1,58 @@
+/**
+ * Copyright Zendesk, Inc.
+ *
+ * Use of this source code is governed under the Apache License, Version 2.0
+ * found at http://www.apache.org/licenses/LICENSE-2.0.
+ */
+
+import React from 'react';
+import styled from 'styled-components';
+import { ISidebarSection } from '../../../layouts/Sidebar';
+import { LG, UnorderedList } from '@zendeskgarden/react-typography';
+import { Link } from '../../../layouts/Root/components/StyledNavigationLink';
+
+const StyledUnorderedList = styled(UnorderedList)`
+  margin: 0 0 ${p => p.theme.space.sm} 0;
+`;
+
+const StyledListItem = styled(UnorderedList.Item)`
+  list-style: none;
+`;
+
+export const OverviewLinks: React.FC<{ nav: ISidebarSection[] }> = ({ nav }) => {
+  const content = nav.map(section => (
+    <StyledUnorderedList key={section.title}>
+      <LG isBold>{section.title}</LG>
+      {section.items && (
+        <StyledUnorderedList>
+          {section.items?.map(group => {
+            if (group.items) {
+              return (
+                <StyledListItem key={group.title}>
+                  {group.title}
+                  <UnorderedList type="disc">
+                    {group.items.map(child => {
+                      return (
+                        <UnorderedList.Item key={child.title}>
+                          <Link to={`${child.id}`}>{child.title}</Link>
```

Comment:
Extra string template exist on `to` prop.

Reply:
VSCode gets upset with me if I remove the literals on the Gatsby `to`: https://share.getcloudapp.com/RBuqvmOv Any idea what might be causing that?

Reply:
Also, if you run `yarn clean` do you still get those export warnings? I'm not getting any in my console and I'm not sure why.

Reply:
> Also, if you run yarn clean do you still get those export warnings? I'm not getting any in my console and I'm not sure why.

I should have clarified, the logs from my latest comment are from the CI build. The warnings don't appear locally anymore. Could be a stale CircleCI cache.